src/App/Components/DisplayArticle/breadCrumb.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BreadCrumb = (props) => {
```

Comment:
I need more explanation on this breadCrumb, I feel like it is not navigation aware my friend @MCFrank16

Reply:
@MCFrank16 any comments on @truestbyheart's note above?